rules/game_process.tex

```diff
@@ -178,33 +190,99 @@ \subsubsection{Indirect Kick - Champions Cup}
   Player 2 (of the red team) kicks the ball again and score.
   This is \textit{not} an indirect kick, the goal does not count and a \texttt{Goal kick} is awarded to the blue team.
+  \item[Example 7:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) kicks directly into the blue goal.
+  The failed-free-kick exception applies, and the goal counts.
+  \item[Example 8:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) kicks the ball 1 meter away.
+  The ball stops, Player 2 kicks again, and scores.
+  The exception does \textit{not} apply for the red team because Player 2 played the ball twice. The goal does not count and a \texttt{Goal kick} is awarded to the blue team.
+  \item[Example 9:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) passes the ball to Player 3 (of the red team), who kicks again and scores.
+  Although the exception doesn't apply because Player 3 played the ball as well, this is still an ordinary indirect kick, so the goal counts.
+  \item[Example 10:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, and Player 2 (of the red team) kicks the ball 1 meter away.
+  Player 5 (of the blue team) kicks the ball, then Player 2 kicks and scores.
+  The exception does \textit{not} apply for the red team because Player 5 also played the ball. The goal does not count and a \texttt{Goal kick} is awarded to the blue team.
+  \item[Example 11:] The blue team is awarded a free kick, but doesn't play the ball. The timer expires, then Player 5 (of the blue team) kicks the ball.
+  Player 2 (of the red team) then kicks the ball and scores.
+  The exception does \textit{not} apply for the red team because a blue-team robot played the ball first. The goal does not count and a \texttt{Goal kick} is awarded to the blue team.
 \end{description}
 
-\subsubsection{Indirect Kick - Challenge Shield}
-\label{sec:indirect_kick_challenger}
+\subsubsection{Fallback mode for Challenge Shield}
+
+This rule aims to allow \textbf{Challenge Shield} teams to play games
+even in the unfortunate case where severe circumstances make them unable to
+field enough robots to play by the default indirect kick rules.
+
+Each Challenge Shield team participating in a game shall declare their operational
+capacity for that game by choosing to play either in \textbf{normal mode} or in \textbf{fallback mode}.
+If at least one team plays in \textit{fallback mode}, the whole game follows different,
+more lenient rules for indirect kicks, detailed in \cref{sec:indirect_kick_old}.
+
+If a team plays in \textit{fallback mode}, they are only allowed to field at most two robots.
```

Comment:
Maybe specify that this doesn't affect the number of robot allowed for the other team ?